lib/ansible/modules/system/firewalld.py

```diff
@@ -98,6 +98,26 @@
       - Whether to run this module even when firewalld is offline.
     type: bool
     version_added: "2.3"
+  direct_rule:
+    description:
+      - 'The firewalld iptables direct rule you would like to enable/disable.'
```

Comment:
We should be adding type-information to all parameters starting with Ansible v2.8.
Also, no need to quote the description, we tend to only quote when it is required by YAML.
```suggestion
      - The firewalld iptables direct rule you would like to enable/disable.
    type: str
```

Reply:
Ah, sorry ... I've been shuffled off on a side project for a while so I'm a little out of the loop on Core on-goings. Thanks, will fix.

Reply:
Please do :-)